apps/nextra/pages/en/build/indexer.mdx

```diff
@@ -38,19 +38,19 @@ Here's an interactive example of how you can query for the Current Coin Balances
 
 <Cards>
   <Card href="./indexer/api/labs-hosted">
-    <Card.Title>Aptos Labs Indexer API</Card.Title>
+    <Card.Title>Aptos Labs Indexer API →</Card.Title>
```

Comment:
lol are we going to keep up with all the arrows?

Reply:
Hahahhah YESSSSS